src/clr-core/icon/shapes/bubbleChart.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { renderIcon } from '../icon.renderer';
```

Comment:
Not a huge deal, but should we name the file `bubble-chart.ts` to be consistent with other file names?

Reply:
Yep.